pavelib/assets.py

```diff
@@ -194,6 +194,17 @@ def get_theme_sass_dirs(system, theme_dir):
             ],
         })
 
+        # now compile theme sass files for certificate
+        if system == 'lms':
+            dirs.append({
+                "sass_source_dir": theme_dir / system / "static" / "certificates" / "sass",
```

Comment:
Perpahs you would like to extract the dir in the same way it is done in lines 167-169 to a variable `certs_sass_dir` or similar.

Reply:
Good suggestion. It will improve readability.